images/libvirtd/libvirtd-ds.yaml

```diff
@@ -12,40 +12,21 @@ spec:
     spec:
       hostNetwork: true
       hostPID: true
+      hostIPC: true
       securityContext:
         runAsUser: 0
       containers:
-        - name: daemon
+        - name: libvirtd
           image: kubevirt/libvirtd:latest
           ports:
             - containerPort: 16509
           securityContext:
             privileged: true
             runAsUser: 0
           volumeMounts:
-            - mountPath: /etc/libvirt
-              name: etc
-            - mountPath: /var/run/libvirt
```

Comment:
Intially we want to keep it ocmpletely isolated. Just for simplicity reasons.

The general concept is: Keep it isolated and self contained.

However, I do see value in at least keeping the runtime informations outside the container, to allow restarting of the container.

Would be a separate patch tho, and I would not do it right now (because it needs some considerations when we containers OS != host OS)